src/components/list-player.js

```diff
@@ -0,0 +1,14 @@
+
+
+const PlayerList = () => {
+
+
+    return (
+        <div>
+            
+        </div>
+    )
+}
+
+// Default Export
+export default PlayerList;
```

Comment:
I think this component may be redundant with the player list stored in the game state in GameProvider.js

Reply:
I agree, I'll omit this